### PR TITLE
Add more explanation to activation scenarios

### PR DESCRIPTION
**Pull request by rakina**
PTAL :)

### CODE REVIEW

File: cheatsheet.md [hunk 1]
```diff
@@ -74,25 +46,42 @@ a `rendersubtreeactivation` signal is fired on the element and the `rendersubtre
...
+* When text in locked subtree gets selected, all of the text's locked ancestors will be activated.
+* Can be disabled with `skip-viewport-activation` or `skip-activation`.
+ 
+### Sequential/tab-order focus navigation
+* When an element is focused by sequential focus navigation (forward or backward),
+the locked ancestors of the focused element will be activated
+* Can disabled with `skip-viewport-activation` or `skip-activation`.
+
+### Find-in-page
+* Find-in-page will find text even if they are in a locked subtree (but will skip locked subtrees with `skip-activation`), counting them in the total match.
```
Comment (vmpstr) [inline cheatsheet.md]: s/they are/it is/

I would maybe refrase as "... even if it is in a locked activatable subtree ...". With possibly a note before this whole block explaining the difference between locked activatable and locked non-activatable

File: cheatsheet.md [hunk 2]
```diff
@@ -74,25 +46,42 @@ a `rendersubtreeactivation` signal is fired on the element and the `rendersubtre
...
-* selection: when an element is selected, it is activated so its children can be
-  selected as well (disabled by `skip-viewport-activation`)
-* focus(): when focus() is called on an element, its locked ancestors are
-  activated (disabled by `skip-activation`)
+### Viewport intersection
+* When a locked element enters the viewport or nearing the viewport (50% margin), it will be activated.
+* Can be disabled with `skip-viewport-activation` or `skip-activation`.
+
+### Selection
+* When text in locked subtree gets selected, all of the text's locked ancestors will be activated.
```
[the remark this answers is not in the document]
Comment (rakina) [reply cheatsheet.md]: Oops yeah we unlock it as it's also part of the selection, changed to "content"

File: cheatsheet.md [hunk 3]
```diff
@@ -7,35 +7,6 @@ and status. Note that this does not go into details on what each component
 represents and why. Instead, this acts as an easy way to map up-to-date concepts
 with possibly out-of-date explainers.
 
-## Revision log
```
[the remark this answers is not in the document]
Comment (rakina) [reply cheatsheet.md]: I think that date is obtainable from the git log?

File: cheatsheet.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (rakina) [reply cheatsheet.md]: oops yeah, changed to "content" instead of "text", is that better?

File: cheatsheet.md (hunk 1 above)
Comment (rakina) [reply cheatsheet.md]: Rephrased a bit & added more explanation.

File: cheatsheet.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (vmpstr) [reply cheatsheet.md]: okay

**Pull request merged**